obspy/io/css/core.py

```diff
@@ -72,6 +72,40 @@ def _is_css(filename):
     return True
 
 
+def _is_nnsa_kb_core(filename):
+    """
+    Checks whether a file is NNSA KB Core waveform data (header) or not.
+
+    :type filename: str
+    :param filename: NNSA KB Core file to be checked.
+    :rtype: bool
+    :return: ``True`` if a NNSA KB Core waveform header file.
+    """
+    # Fixed file format.
+    # Tests:
+    #  - the length of each line (287 chars)
+    #  - two epochal time fields
+    #    (for position of dot and if they convert to UTCDateTime)
+    #  - supported data type descriptor
+    try:
+        with open(filename, "rb") as fh:
+            lines = fh.readlines()
+            # check for empty file
+            if not lines:
+                return False
+            # check every line
+            for line in lines:
+                assert(len(line.rstrip(b"\n\r")) == 287)
+                assert(line[27] == b".")
+                UTCDateTime(float(line[16:33]))
+                assert(line[73] == b".")
+                UTCDateTime(float(line[62:79]))
+                assert(line[144:146] in DTYPE)
+    except:
```

Comment:
Blanket exception handling is hiding whatever the error is on Python 3, since this function is returning `False` instead of `True`.